From 18b2e3c4b014528707f9b4192cf50b382ab2d7e5 Mon Sep 17 00:00:00 2001
From: "M, Kumar K" <kumar.k.m@intel.com>
Date: Tue, 4 Dec 2018 16:19:53 +0530
Subject: [PATCH] Fix Audio CTS issue in mediaplayer

Revert "Fix the flicker on single touch"

This reverts commit 60015279975f04d5be2012cd34fad58a1f400ad3.

This patch is creating a CTS regression in audio.

Change-Id: I3fd3c7476a9a3dc9315f789c20e34c75248c90f3
---
 media/libmedia/include/media/mediaplayer.h | 1 -
 media/libmedia/mediaplayer.cpp             | 7 -------
 2 files changed, 8 deletions(-)

diff --git a/media/libmedia/include/media/mediaplayer.h b/media/libmedia/include/media/mediaplayer.h
index 3b235ee..2335c5a 100644
--- a/media/libmedia/include/media/mediaplayer.h
+++ b/media/libmedia/include/media/mediaplayer.h
@@ -294,7 +294,6 @@ private:
     int                         mCurrentPosition;
     MediaPlayerSeekMode         mCurrentSeekMode;
     int                         mSeekPosition;
-    int                         mSeekTime;
     MediaPlayerSeekMode         mSeekMode;
     bool                        mPrepareSync;
     status_t                    mPrepareStatus;
diff --git a/media/libmedia/mediaplayer.cpp b/media/libmedia/mediaplayer.cpp
index e53361b..26908e5 100644
--- a/media/libmedia/mediaplayer.cpp
+++ b/media/libmedia/mediaplayer.cpp
@@ -60,7 +60,6 @@ MediaPlayer::MediaPlayer()
     mCurrentPosition = -1;
     mCurrentSeekMode = MediaPlayerSeekMode::SEEK_PREVIOUS_SYNC;
     mSeekPosition = -1;
-    mSeekTime = -1;
     mSeekMode = MediaPlayerSeekMode::SEEK_PREVIOUS_SYNC;
     mCurrentState = MEDIA_PLAYER_IDLE;
     mPrepareSync = false;
@@ -585,12 +584,6 @@ status_t MediaPlayer::seekTo_l(int msec, MediaPlayerSeekMode mode)
 
 status_t MediaPlayer::seekTo(int msec, MediaPlayerSeekMode mode)
 {
-    if (mSeekTime == msec) {
-        ALOGW("Duplicate seek time: %d", msec);
-        return NO_ERROR;
-    } else {
-        mSeekTime = msec;
-    }
     mLockThreadId = getThreadId();
     Mutex::Autolock _l(mLock);
     status_t result = seekTo_l(msec, mode);
-- 
1.9.1

